Documentation/conversion.md

```diff
@@ -1,61 +1,41 @@
 ---
-title: Conversion in Azure Remote Rendering asset tool
-description: Workflow to convert a model in ARRT
-author: mafranc
-ms.author: mafranc
-ms.date: 03/23/2020
+title: Converting Models with ARRT
+description: Describes how to convert a 3D model with the Azure Remote Rendering Toolkit
+author: jakras
+ms.author: jakras
+ms.date: 12/23/2021
 ms.topic: article
 ---
 
-# Conversion in Azure Remote Rendering asset tool
+# Converting Models with ARRT
 
-![Conversion panel](media/conversion.png)
-To access the conversion environment, click on the button on the top toolbar.
-To convert a file, click on "New". Now the view is split between a conversion list on the left side, and the conversion configuration panel on the right side. Follow these steps to configure your new conversion:
-
-## Input model selection
-
-If you click on the button "Select" on the right of the input field, a panel will show the content of your blob storage. To choose the model to convert, first choose the blob container, then select an input model.
-
-The blob list visualization has two modes:
-
-* When **show all models** button is selected, all of the models will be shown in a flat list. If a directory is selected, then the list will be filtered with only the models in that directory or subdirectories. The list shows just the valid 3d file input formats (FBX, GLFT, GLB).
-![Show all models](media/standardbloblist.png)
-* When **show all models** is not selected then the list will show only the content of the selected directory, with all of its blobs and subdirectories. You can double-click on the folders to navigate into them, like a file explorer, and find your model.
-![Show only directory](media/directorybloblist.png)
+Under the *Conversion* tab you can convert a model from a source asset format to the internal `.arrAsset` format needed by Azure Remote Rendering. ARRT simply gathers the information and forwards it to the ARR model conversion service. Thus all conversions run in the background in the cloud. Details on supported file formats, pricing and so on, [can be found here](https://docs.microsoft.com/azure/remote-rendering/how-tos/conversion/model-conversion).
 
-You can select an input model from the list by double-clicking the row, or selecting it and pressing "Select input".
+## Starting a Conversion
```

Comment:
sentence capitalization in headlines (unless you want to use capital 'Conversion' everywhere?)

Reply:
...and in all other headlines as well